spec/lib/api/v3/work_packages/work_package_representer_spec.rb

```diff
@@ -200,21 +200,21 @@
     end
 
     describe 'work_package' do
-      it { should have_json_path('spentTime') }
+      it { is_expected.to have_json_path('spentTime') }
 
-      it { should_not have_json_path('spentHours') }
+      it { is_expected.not_to have_json_path('spentHours') }
 
-      it { should_not have_json_path('overallCosts') }
+      it { is_expected.not_to have_json_path('overallCosts') }
 
       describe 'embedded' do
-        it { should_not have_json_path('_embedded/costObject') }
+        it { is_expected.not_to have_json_path('_embedded/costObject') }
 
-        it { should_not have_json_path('_embedded/summarizedCostEntries') }
+        it { is_expected.not_to have_json_path('_embedded/summarizedCostEntries') }
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.